src/components/Trade/hooks/useSwapper/useSwapperV2.tsx

```diff
@@ -144,6 +174,7 @@ export const useSwapper = () => {
       return bestTradeSwapper.buildTrade({
         ...buildTradeCommonArgs,
         chainId: sellAssetChainId,
+        bip44Params: sellAccountBip44Params,
```

Comment:
accountType?

Reply:
Non-utxo branch, this will use the `CommonTradeInput` interface (without `accountType`) which is correct